plugins/importexport/native/filter/NativeXmlArticleGalleyFilter.inc.php

```diff
@@ -74,6 +74,7 @@ function handleElement($node) {
 			if ($newSubmissionFileId) $addSubmissionFile = true;
 		}
 		$representation = parent::handleElement($node);
+		$representation->setData('urlPath', $node->getAttribute('url_path'));
```

Comment:
Try adding this line at `NativeXmlRepresentationFilter`, which is the parent class. That way your change will go into the PKP-LIB project, effecting all applications (OJS, OMP, OPS). Moreover, the `url_path` attribute is declared here (https://github.com/pkp/pkp-lib/blob/master/plugins/importexport/native/pkp-native.xsd#L224) which resides at PKP-LIB project.